airbyte-integrations/bases/source-acceptance-test/setup.py

```diff
@@ -13,7 +13,7 @@
     "inflection~=0.5",
     "pdbpp~=0.10",
     "pydantic~=1.6",
-    "pytest~=6.1",
+    "pytest~=6.2",
```

Comment:
```suggestion
    "pytest~=6.2,>=6.2.5"
```
Optional: In places where we don't need to pin a specific version, we can ensure that the downloaded version is compatible by specifying the minimum version as well as the `~=` version. 

Your fix makes it so that anyone who runs a new install gets the correct version -- pinning the minimum would make it so that someone who already has a 6.2.3 version, for example, gets the upgrade too if they try to re-install as a first step to debugging this issue. 

Since most people who run into this were probably on a 6.1 version, it's probably fine not to do so, just wanted to bring it up.

Reply:
I *think* the minversion takes care of that for us...

Reply:
Oh! Yes, missed that. I think it should do it too.